.sync.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+mock_with: ':rspec'
+
+.gitlab-ci.yml:
+  delete: true
+appveyor.yml:
+  delete: true
+
+.travis.yml:
+  user: MiamiOH
+  secure: "RMGe+qbnUvk2tM7bXlLObXjlNB+C1cyHOU3FGJS0GdESVkdI6Wne66Y6tI5XPlWkI4UA0YWU/38fJllLuOdbKooNg5XOefocc5PQbmDbi8A7jRYMiZx7J9lblJKqb0HspmVsrWzz1Ss8l/UmkQWWAUAHU4yHTPquqFQV19CZytE="
```

Comment:
We need too generate a secure key that is specific to this repo with the `travis secure` command